docs/DeveloperGuide.md

```diff
@@ -239,6 +240,47 @@ _{more aspects and alternatives to be added}_
 _{Explain here how the data archiving feature will be implemented}_
 
 
+### Listing Events
+
+The List Events feature allows users to enter `listEvents` and update the UI display for events, showing a list of all 
+events sorted in their current order inside `ModelManager`. In the future, the command will take in a parameter,
+which specifies the field that the events list can be permanently sorted by.
+
+### Implementation
+The List Events feature is facilitated by `ListEventsCommand` which extends from Command. Additionally, it implements
+the following operation:
+
+* `ModelManager#updateFilteredEventList`  — Updates the predicate inside `ModelManager`'s filtered event list 
+to modify and sort which and how events are shown.
+
+This operation is exposed in the `Model` interface as the method `Model#updateFilteredEventList`.
+
+The following sequence diagram shows how the listEvents operation works.
+
+![ListEventsSequenceDiagram](images/ListEventsSequenceDiagram.png)
```

Comment:
I think the input argument `listEvents` for the `execute(listEvents)` and `parseCommand(listEvents)` methods should be wrapped in quotes to indicate that it is a string. i.e. `execute("listEvents")` and `parseCommand("listEvents")`